opentelemetry-sdk/src/metrics/internal/exponential_histogram.rs

```diff
@@ -100,9 +100,20 @@ impl<T: Number> ExpoHistogramDataPoint<T> {
             if (self.scale - scale_delta as i8) < EXPO_MIN_SCALE {
                 // With a scale of -10 there is only two buckets for the whole range of f64 values.
                 // This can only happen if there is a max size of 1.
-                opentelemetry::global::handle_error(MetricsError::Other(
-                    "exponential histogram scale underflow".into(),
-                ));
+
+                // TODO - to check if this should be logged as an error if this is auto-recoverable.
+                otel_debug!(
+                    name: "ExponentialHistogramDataPoint.Scale.Underflow",
+                    current_scale = self.scale,
+                    scale_delta = scale_delta,
+                    max_size = self.max_size,
+                    min_scale = EXPO_MIN_SCALE,
+                    record_min_max = self.record_min_max,
```

Comment:
these configs are too much information, so better skip it.

Reply:
I think max_size, min_scale, value and message should be kept, rest can be removed.